### PR TITLE
feature/AT-UN-UI-14 Create dashboard page

### PR DESCRIPTION
**Pull request by MajoYanez**
Creation of dashboard to view all courses.
@fasaga

### CODE REVIEW

File: src/routes/course.routes.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 const router = require("express").Router();
+const { getAllCourses } = require ('../controllers/TestController'); 
+const { renderCourses } = require ('../controllers/dashboard.controller');
 
 /* Routes */
 
+//Get All Courses
```
Comment (fasaga) [inline src/routes/course.routes.js]: Remove these 2 lines, we do not need them

File: src/routes/course.routes.js [hunk 2]
```diff
@@ -1,5 +1,13 @@
 const router = require("express").Router();
+const { getAllCourses } = require ('../controllers/TestController');
```
Comment (fasaga) [inline src/routes/course.routes.js]: Remove this line

File: src/controllers/dashboard.controller.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const { getAllCourses } = require("../services/course.service");
+
+/*This is a test controller for course service*/
```
Comment (fasaga) [inline src/controllers/dashboard.controller.js:3]: Remove this line

File: src/controllers/courses.controller.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+const { getAllCourses } = require("../services/course.service");
```
Comment (fasaga) [inline src/controllers/courses.controller.js:1]: Leave this file empty, do not forget to remove also the references to its functions

File: src/routes/course.routes.js (hunk 2 above)
Comment (at-internship) [inline src/routes/course.routes.js]: should be test.controller.js

File: src/routes/course.routes.js [hunk 5]
```diff
@@ -1,5 +1,13 @@
 const router = require("express").Router();
+const { getAllCourses } = require ('../controllers/TestController'); 
+const { renderCourses } = require ('../controllers/dashboard.controller');
 
 /* Routes */
 
+//Get All Courses 
+router.get("/course",getAllCourses);
```
Comment (at-internship) [inline src/routes/course.routes.js]: router.get("/course", getAllCourses);

**Review by at-internship — CHANGES_REQUESTED** [review]
Please verify, we have some comments

**Review by fasaga — APPROVED** [review]

**Comment by fasaga** [comment]
@at-internship Could you please approve this pull request?

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**